python/amici/ode_export.py

```diff
@@ -1165,7 +1165,7 @@ def _generateSymbol(self, name):
                 for comp in getattr(self, component)
             ])
             self._strippedsyms[name] = sp.Matrix([
-                sp.Symbol(comp.get_name(), real=True)
```

Comment:
Could you add a comment why this is not supposed to be real? Otherwise I might change it again... Not intuitive for me.

Reply:
Symbols have globally unique ids depending on their name and assumptions. To be able to differentiate we need to differentiate wrt to symbols with the exact same id, so we need to create them with exactly same assumptions as they were originally created.

There are probably better ways to solve this but that snippet adresses an issue with an unofficial branch that hasn't made it's way into master yet and is still subject to change so I don't want to spend too much effort on this until we are certain this is really the way things are going to be like in the future.

Reply:
First part is clear, yes. That's why I made everything real.

Can't just make other symbols real as well?

As long as the SBML test suite still pases, fine for me ;). Comment in the code as reminder would still be good.

Reply:
We are not the ones creating these symbolic variables here ;). We are just calling `sp.Symbol` as a way to get access to the ones to that were (possibly) created by pysb.